GDJS/Runtime/runtimeobject.js

```diff
@@ -113,7 +117,12 @@ gdjs.RuntimeObject.prototype.getElapsedTime = function(runtimeScene) {
  * @param {gdjs.RuntimeScene} runtimeScene The gdjs.RuntimeScene the object belongs to.
  */
 gdjs.RuntimeObject.prototype.update = function(runtimeScene) {
-    //Nothing to do.
+    //Update timers
+    for (var name in this._timers.items) {
+        if (this._timers.items.hasOwnProperty(name)) {
+            this._timers.items[name].updateTime(this.getElapsedTime(runtimeScene));
```

Comment:
I think you can store `this.getElapsedTime(runtimeScene) ` in a temporary variable at the beginning of `update` to avoid computing this again and again.